From 4f1c71f3f3b2d959d314c747550370b74e885a9b Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@nxp.com>
Date: Tue, 18 Jul 2017 20:40:05 +0800
Subject: [PATCH 2228/5242] MLK-16013-3 usb: dwc3: of-simple: add imx8mq usb
 support

commit  9335a952440511a7a9a01c6dc64ac92c0c4262b3 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add i.mx8mq USB3 phy compatible string for dwc3-of-simple driver.

Reviewed-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: Li Jun <jun.li@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/usb/dwc3/dwc3-of-simple.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/usb/dwc3/dwc3-of-simple.c b/drivers/usb/dwc3/dwc3-of-simple.c
index dbeff5e..5d33e2a 100644
--- a/drivers/usb/dwc3/dwc3-of-simple.c
+++ b/drivers/usb/dwc3/dwc3-of-simple.c
@@ -216,6 +216,7 @@ static int dwc3_of_simple_runtime_resume(struct device *dev)
 	{ .compatible = "amlogic,meson-axg-dwc3" },
 	{ .compatible = "amlogic,meson-gxl-dwc3" },
 	{ .compatible = "allwinner,sun50i-h6-dwc3" },
+	{ .compatible = "fsl, imx8mq-dwc3" },
 	{ /* Sentinel */ }
 };
 MODULE_DEVICE_TABLE(of, of_dwc3_simple_match);
-- 
1.7.9.5

